Rakefile

```diff
@@ -11,7 +11,10 @@ RuboCop::RakeTask.new
 
 task :default => [:spec, :rubocop, :cucumber]
 
-if ENV['TRAVIS']
+# Skip Ruby 2.4 until coveralls supports simplecov (>= 0.13.0)
+# for Ruby 2.4 compatibility.
+# https://rubygems.org/gems/coveralls
+if ENV['TRAVIS'] && RUBY_VERSION < '2.4'
   ENV['SIMPLECOV']  = 'ci'
```

Comment:
I am not sure about this. What is the advantage in skipping 2.4 coverage? I think having ruby 2.4.0 in allowed failures is not great, but at least is more explicit.

Also, as far as I understand there is also PR open on `coveralls` to fix this: https://github.com/lemurheavy/coveralls-ruby/pull/117